actions/invite_actions.jsx

```diff
@@ -37,9 +37,10 @@ export function sendMembersInvites(teamId, users, emails) {
             } catch (e) {
                 response = {error: localizeMessage('invite.members.unable-to-add-the-user-to-the-team', 'Unable to add the user to the team.')};
             }
+            const errors = response.data.errors;
```

Comment:
probably something like `response.data.errors || {};` what happens if no error is returned?